### ISSUE TITLE
Background exec

### ISSUE THREAD

**Issue by louia** [wallforfry/dart_mercure#4]
Is there any solution to have a background execution for have mercure + notification + background subscription to mercure topic ?

**Comment by wallforfry** [comment]
Hi @louia !
It looks like the only way to implement a background service is to use native code for iOS and Android.

[This plugin](https://pub.dev/packages/background_fetch) allow you wake up your app periodically but it doesn't match with Mercure usage.

According to [Flutter documentation](https://flutter.dev/docs/development/packages-and-plugins/background-processes), it's possible to use a native service to wake up a small part of flutter app.
I think the best way is to create a PR that implements for both OSs a mercure listener and that pass received data in background to the woke up app

**Comment by louia** [comment]
Yes i think is a good idea. If I have the time i will do that but if anyone can start or do. (Oh un français !)
